src/controller/python/chip/clusters/Attribute.py

```diff
@@ -585,10 +601,13 @@ def Shutdown(self):
             print("Subscription was already terminated previously!")
             return
 
-        handle = chip.native.GetLibraryHandle()
-        builtins.chipStack.Call(
-            lambda: handle.pychip_ReadClient_Abort(
-                self._readTransaction._pReadClient, self._readTransaction._pReadCallback))
+        # Abort only when no error occurs.
+        # If an error occurs, pReadClient has been deleted internally.
+        if self._readTransaction._resultError is None:
```

Comment:
Please open an issue showing what is currently failing, so that it's clear what is being fixed.

Reply:
refer to the first point of comment